be/src/olap/compaction.cpp

```diff
@@ -186,4 +186,17 @@ OLAPStatus Compaction::check_correctness(const Merger& merger) {
     return OLAP_SUCCESS;
 }
 
+OLAPStatus Compaction::check_disk_capacity() {
+    int64_t incoming_data_size = 0;
```

Comment:
only in SegmentWriter will be ok. All writers only go through the SegmentWriter.